lib/server.js

```diff
@@ -1169,5 +1170,54 @@ function _finishReqResCycle(req, res, route, err) {
     }
 
     // decrement number of requests
-    self._unfinishedRequests--;
+    self._inflightRequests--;
+};
+
+
+
+/**
+ * Return debug information about the server.
+ * @public
+ * @method debugInfo
+ * @returns {Object}
+ */
+Server.prototype.debugInfo = function debugInfo() {
```

Comment:
`toJSON()` isn't correct. This gives us the internal state of the object -- whereas `JSON` is merely the way in which we serialize that state -- which we could elect to change. I'm open to other names.